conf/smart-default.xml

```diff
@@ -375,4 +375,20 @@
       Warning: it requires admin privileges for ssm user to be able to configure the db.
     </description>
   </property>
+
+  <property>
+    <name>smart.ignore.path.templates</name>
+    <value></value>
+    <description>
+      Comma-separated list of regex templates of HDFS paths to be completely ignored by SSM.
+    </description>
+  </property>
+
+  <property>
+    <name>smart.internal.path.templates</name>
+    <value>.*/\..*,.*/__.*,.*_COPYING_.*</value>
```

Comment:
Do we plan to do the same for cover dirs? (Introduce templates etc)

Reply:
As far as I know, now we don't have such plans